src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -93,14 +93,16 @@ protected Properties getProperties() {
         result.put("jenkins_base", configuration.getJenkinsBasePath());
         result.put("build_project", configuration.getProjectName());
         result.put("build_number", configuration.getBuildNumber());
-        int previousBuildNumber = -1;
-        try {
-            previousBuildNumber = Integer.parseInt(configuration.getBuildNumber());
-            previousBuildNumber--;
-        } catch (NumberFormatException e) {
-            LOG.error("Could not parse build number: {}.", configuration.getBuildNumber(), e);
+        if (configuration.isRunWithJenkins()){
+            int previousBuildNumber = -1;
+            try {
+                previousBuildNumber = Integer.parseInt(configuration.getBuildNumber());
+                previousBuildNumber--;
+            } catch (NumberFormatException e) {
+                LOG.error("Could not parse build number: {}.", configuration.getBuildNumber(), e);
+            }
+            result.put("build_previous_number", previousBuildNumber);
```

Comment:
In that case the conversion string 2 integer is useless. This is the only one line that should be used. Pass what ever you have as the String and make checking here https://github.com/damianszczepanik/cucumber-reporting/blob/master/src/main/resources/templates/topMenu.vm#L9